source/form.html

```diff
@@ -148,32 +150,31 @@ <h2 class="visually-hidden">Анкета</h2>
         <fieldset class="form__input form__input--comment-user">
           <legend class="form__feildset-title title title--large">Комментарий</legend>
           <div class="form__item">
-            <textarea class="form__item-description" style="width: 100%; height: 100%;">Расскажите обо всех повадках кота</textarea>
+            <textarea class="form__item-description"
+              style="width: 100%; height: 100%;">Расскажите обо всех повадках кота</textarea>
           </div>
         </fieldset>
 
         <fieldset class="form__input form__input--add-user add-user">
           <legend class="form__feildset-title title title--large">Дополнительно</legend>
-          <div class="form__list-wrapper">
-            <ul class="form__list form__list--add">
-              <li class="item-info item-info--add">
-                <input type="checkbox" name="sweetener" id="sweetener-field" checked>
-                <label for="sweetener-field">сахарозаменитель</label>
-              </li>
-              <li class="item-info item-info--add">
-                <input type="checkbox" name="water" id="water-field">
-                <label for="sweetener-field">питьевая вода</label>
-              </li>
-              <li class="item-info item-info--add">
-                <input type="checkbox" name="milk" id="milk-field">
-                <label for="sweetener-field">молоко</label>
-              </li>
-              <li class="item-info item-info--add">
-                <input type="checkbox" name="vitamins" id="vitamins-field">
-                <label for="sweetener-field">витамины</label>
-              </li>
-            </ul>
-          </div>
+          <ul class="form__list form__list--add input-list">
+            <li class="input-list__item item-info">
+              <input type="checkbox" name="sweetener" id="sweetener-field" checked>
+              <label for="sweetener-field">сахарозаменитель</label>
+            </li>
+            <li class="input-list__item item-info">
+              <input type="checkbox" name="water" id="water-field">
+              <label for="sweetener-field">питьевая вода</label>
+            </li>
+            <li class="input-list__item item-info">
+              <input type="checkbox" name="milk" id="milk-field">
+              <label for="sweetener-field">молоко</label>
+            </li>
+            <li class="input-list__item item-info--add">
```

Comment:
```suggestion
            <li class="input-list__item item-info item-info--add">
```
Модификатор не используется в одиночку